libs/community/langchain_community/embeddings/sparkllm.py

```diff
@@ -5,25 +5,23 @@
 import logging
 from datetime import datetime
 from time import mktime
-from typing import Any, Dict, List, Optional
+from typing import Any, Dict, List, Literal, Optional
 from urllib.parse import urlencode
 from wsgiref.handlers import format_date_time
 
 import numpy as np
 import requests
 from langchain_core.embeddings import Embeddings
-from langchain_core.pydantic_v1 import BaseModel, SecretStr, root_validator
+from langchain_core.pydantic_v1 import BaseModel, Field, SecretStr, root_validator
 from langchain_core.utils import convert_to_secret_str, get_from_dict_or_env
 from numpy import ndarray
 
-# Used for document and knowledge embedding
-EMBEDDING_P_API_URL: str = "https://cn-huabei-1.xf-yun.com/v1/private/sa8a05c27"
-# Used for user questions embedding
-EMBEDDING_Q_API_URL: str = "https://cn-huabei-1.xf-yun.com/v1/private/s50d55a16"
+# Used for embedding
+EMBEDDING_API_URL: str = "https://emb-cn-huabei-1.xf-yun.com/"
```

Comment:
should we make this configurable as well?

Reply:
Should I add 'base_url' as a class attribute?

Reply:
Done